compiler/src/dotty/tools/dotc/core/tasty/TreeBuffer.scala

```diff
@@ -3,8 +3,11 @@ package dotc
 package core
 package tasty
 
-import util.Util.{bestFit, dble}
-import TastyBuffer.{Addr, NoAddr, AddrWidth}
+import dotty.tools.tasty.util.Util.dble
+import dotty.tools.tasty.TastyBuffer
+import TastyBuffer.{ Addr, NoAddr, AddrWidth }
```

Comment:
Try to avoid changing the import format we use (no spaces around the list of imports)

Reply:
I've fixed it